services/brig/brig.integration.yaml

```diff
@@ -55,8 +55,9 @@ emailSMS:
   # or you can use SMTP directly (blacklisting of email/phone must be otherwise handled by
   # the operator).
   email:
-    sesQueue: integration-brig-events
-    sesEndpoint: http://localhost:4569 # https://email.eu-west-1.amazonaws.com
+    noEndpoint: false
```

Comment:
double negation noEndpoint: false = endpoint: true which is confusing. Rename this option, or reverse the boolean logic?